### PR TITLE
Update HepMC3 to 3.2.6

### PR DESCRIPTION
**Pull request by andriish**
### Have you:

- [ ] Run `brew audit --strict --online <formula>` (where `<formula>` is the name of the formula you're submitting) and corrected all errors?
- [ ] Built your formula locally prior to submission with `brew install <formula>`?

### Updates to existing formula: have you

- [ x ] Checked that the tests still pass?

### CODE REVIEW

File: Formula/hepmc3.rb [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
   desc "C++ event record for Monte Carlo generators"
   homepage "https://hepmc.web.cern.ch/"
-  url "https://hepmc.web.cern.ch/hepmc/releases/HepMC3-3.2.5.tar.gz"
-  sha256 "cd0f75c80f75549c59cc2a829ece7601c77de97cb2a5ab75790cac8e1d585032"
-  license "GPL-3.0-or-later"
-  revision 2
+  url "https://gitlab.cern.ch/hepmc/HepMC3/-/archive/3.2.6/HepMC3-3.2.6.tar.gz"
+  sha256 "648a3725a772e3dc26eaba0fe3c2f941a38d2b39679cea64413a2ae49ad946b0"
+  license "LGPL-3.0-or-later"
+  revision 1
```
Comment (davidchall) [inline Formula/hepmc3.rb:7]: This line should be removed altogether (revision is only needed when a formula changes but the version remains the same).
```suggestion
```

File: Formula/hepmc3.rb [hunk 2]
```diff
@@ -13,18 +13,17 @@ class Hepmc3 < Formula
 
   bottle do
     root_url "https://ghcr.io/v2/davidchall/hep"
-    sha256 cellar: :any, monterey: "d2273c38945cc73c136c961213f3b710caf1b10293be65fffbafeeb8ffc05f72"
```
Comment (davidchall) [inline Formula/hepmc3.rb:16]: You shouldn't need to edit these lines. They will be replaced by the merge bot that runs after you add the "pr-pull" label.

File: Formula/hepmc3.rb (hunk 1 above)
Comment (andriish) [reply Formula/hepmc3.rb]: Done.

File: Formula/hepmc3.rb (hunk 2 above)
Comment (andriish) [reply Formula/hepmc3.rb:16]: Yes.

**Comment by andriish** [comment]
Hi @davidchall,

the soversion of one library was changed, so that is why there are failures.

This should be fine and it is expected, right?

Andrii

**Comment by davidchall** [comment]
Hi @andriish - I'm currently traveling, but will take a look upon my return.

**Pull request merged**